rmf_demo_assets/models/TinyRobot/model.config

```diff
@@ -10,6 +10,6 @@
   </author>
 
   <description>
-    Ubiquity's magni
+    Ubiquity's tinyRobot

```

Comment:
Could you change the description to remove `Ubiquity` as well? That is the company name.

Reply:
Sorry forgot about this. Changed!